src/styles/common/_buttons.scss

```diff
@@ -86,14 +86,22 @@
     }
 }
 
+.btn-plain {
+    margin: 0;
+    padding: 0;
+
+    // prevent default .btn behavior of adding space between buttons
+    & + & {
+        margin: 0;
+    }
+}
+
 .btn-plain,
 .btn-plain:hover,
 .btn-plain:active {
     border: none;
```

Comment:
Can these live in the base class as well? Or do they get set by the browser on hover/active?

Reply:
I don't know and I don't know why :focus doesn't get it, so I am assuming there is a reason to limit the scope of changes